tests/robot-tests/tests/general_public/methodologies_page.robot

```diff
@@ -30,6 +30,27 @@ Validate page contents
     ...    Pupil absence statistics: methodology
     ...    %{PUBLIC_URL}/methodology/pupil-absence-in-schools-in-england
 
+Navigate to /methodology page (prod)
```

Comment:
There is no need to repeat this test case - you can have it run once against the environments you want, since this doesn't depend on test data and should be the same across environments.